<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>expiration</class>
 <widget class="QWidget" name="expiration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>유통기한 관리</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>유통기한 경과 물품</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>841</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>날짜 범위:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="dateFrom">
      <property name="displayFormat">
       <string>yyyy-MM-dd</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>~</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="dateTo">
      <property name="displayFormat">
       <string>yyyy-MM-dd</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSearch">
      <property name="text">
       <string>검색</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>841</width>
     <height>481</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>839</width>
      <height>479</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnMoreItems">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>841</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>더 많은 항목 보기</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>